zerver/lib/test_fixtures.py

```diff
@@ -104,29 +132,27 @@ def database_exists(self) -> bool:
         except OperationalError:
             return False
 
-    def is_current(self) -> bool:
+    def needs_rebuild(self) -> bool:
```

Comment:
I think this might be better named `database_inputs_changed` -- we have another `needs_rebuild` case from migrations having been removed, below, that it's worth distinguishing this from.

Reply:
I went with `files_or_settings_have_changed`, but if you prefer `database_inputs_changed`, please just change it for me.   (I ended up including settings in the new push.)